packages/wonder-blocks-tooltip/components/tooltip.integration.test.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+import * as React from "react";
+
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+import Tooltip from "./tooltip.js";
+import TooltipBubble from "./tooltip-bubble.js";
+import TooltipAnchor from "./tooltip-anchor.js";
+import {TooltipDisappearanceDelay} from "../util/constants.js";
+
+describe("tooltip integration tests", () => {
+    beforeEach(() => {
+        unmountAll();
+        jest.useFakeTimers();
+        // jest.clearAllMocks().resetModules();
+    });
+
+    it("should set timeoutId be null when TooltipBubble is active", () => {
+        const wrapper = mount(
+            <Tooltip content="hello, world">an anchor</Tooltip>,
+        );
+
+        wrapper.setState({active: true});
+
+        const bubbleWrapper = wrapper.find(TooltipBubble);
+        expect(bubbleWrapper.length).toEqual(1);
+
+        expect(wrapper).toHaveState("timeoutID", null);
+    });
+
+    //TODO: Fix, this test interferes with the next one somehow
+    it.skip("should set a timeout on mouseleave on TooltipAnchor", () => {
+        const wrapper = mount(
+            <Tooltip content="hello, world">an anchor</Tooltip>,
+        );
+
+        wrapper.setState({active: true});
+
+        const bubbleWrapper = wrapper.find(TooltipBubble);
+        expect(bubbleWrapper.length).toEqual(1);
+
+        const anchorWrapper = wrapper.find(TooltipAnchor);
+        expect(anchorWrapper.length).toEqual(1);
```

Comment:
I'd remove these expects for `bubbleWrapper` and `anchorWrapper` from this test since this test is specifically looking for `timeoutID`.